app/util/searchUtils.js

```diff
@@ -822,3 +885,23 @@ export const withCurrentTime = (getStore, location) => {
     },
   };
 };
+
+export function reverseGeocode(opts, config) {
+  if (!config.URL.GEOCODING_BASE_URL.includes('digitransit')) {
+    const at = `${opts['point.lat']},${opts['point.lon']}`;
+    return getJson(
+      `${config.URL.GEOCODING_BASE_URL}/geocoder/v1/reverse`,
+      { at },
+      {
+        Accept: 'application/json',
```

Comment:
Since this gets repeated in getJson default params, perhaps merge extra headers with this in that function instead